docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -6,15 +6,14 @@ import PageContext from 'docs/src/modules/components/PageContext';
 import { convertProductIdToName } from 'docs/src/modules/components/AppSearch';
 
 export default function AppFrameBanner() {
-  if (!FEATURE_TOGGLE.enable_docsnav_banner) {
-    return null;
-  }
-
-  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
This was better no? FEATURE_TOGGLE is a global that never changes.

Reply:
Maybe, although it felt better with one less comment disabling rules of hooks. I'll change it back and add a comment about it.